src/secp256k1.c

```diff
@@ -413,6 +413,54 @@ static SECP256K1_INLINE void buffer_append(unsigned char *buf, unsigned int *off
     *offset += len;
 }
 
+/* Initializes SHA256 with fixed midstate. This midstate was computed by applying
+ * SHA256 to SHA256("BIPSchnorrDerive")||SHA256("BIPSchnorrDerive"). */
```

Comment:
```suggestion
 * SHA256 to SHA256("BIP340Derive")||SHA256("BIP340Derive"). */
```

Reply:
Please stop spamming review comments until you understand the code and how it relates to the spec.

Reply:
Ignoring the changes itself,  I may be worth pointing out that it's possible to add a bunch of commits in one shot: 
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review
This creates only a single (email) notification.